src/main/java/com/ericsson/ei/controller/model/Subscription.java

```diff
@@ -212,7 +225,7 @@ public boolean equals(Object other) {
             return false;
         }
         Subscription rhs = ((Subscription) other);
-        return new EqualsBuilder().append(requirements, rhs.requirements).append(created, rhs.created).append(restPostBodyMediaType, rhs.restPostBodyMediaType).append(notificationType, rhs.notificationType).append(userName, rhs.userName).append(aggregationtype, rhs.aggregationtype).append(notificationMessageKeyValues, rhs.notificationMessageKeyValues).append(password, rhs.password).append(repeat, rhs.repeat).append(subscriptionName, rhs.subscriptionName).append(notificationMeta, rhs.notificationMeta).append(additionalProperties, rhs.additionalProperties).append(ldapUserName, rhs.ldapUserName).isEquals();
+        return new EqualsBuilder().append(requirements, rhs.requirements).append(created, rhs.created).append(restPostBodyMediaType, rhs.restPostBodyMediaType).append(notificationType, rhs.notificationType).append(userName, rhs.userName).append(aggregationtype, rhs.aggregationtype).append(notificationMessageKeyValues, rhs.notificationMessageKeyValues).append(password, rhs.password).append(repeat, rhs.repeat).append(subscriptionName, rhs.subscriptionName).append(notificationMeta, rhs.notificationMeta).append(authenticationType, rhs.authenticationType).append(additionalProperties, rhs.additionalProperties).append(ldapUserName, rhs.ldapUserName).isEquals();
```

Comment:
May you reformat this part and try to see if it will break on .append, so each .apend will be in a new laine :)

Reply:
Matias asked the same, this is a generated class.